zerver/lib/subscription_info.py

```diff
@@ -811,21 +814,60 @@ def get_stream_id(sub_dict: RawSubscriptionDict) -> int:
             subscribed_stream_ids,
         )
 
-        # Eventually "partial subscribers" will return (at minimum):
-        # (1) all subscriptions for recently active users (for the buddy list)
-        # (2) subscriptions for all bots
-        #
-        # For now, we're only doing (2).
-        send_partial_subscribers = include_subscribers == "partial"
+        # `partial_subscribers` returns:
+        #  * Any subscriber that would show up in `buddy_data.get_filtered_user_id_list`,
+        #    which determines who is shown in the buddy list. This ensures we show them
+        #    in the correct section. We should be careful to keep this code in sync with
+        #    that buddy list logic.
+        #  * Subscriptions for all bots, so that we can calculate human subscriber
+        #    counts on the client.
         partial_subscriber_map: dict[int, list[int]] = dict()
+
+        # This constant should stay in sync with prescence.ts
+        # (send it from the server?)
+        BIG_REALM_COUNT = 250
```

Comment:
Would it make sense to send this from the server (and do that in a prep commit)?

Reply:
What is its meaning? I'm not sure why it needs to be shared with the client.

Reply:
Check out `BIG_REALM_COUNT` in `presence.ts` -- it's already there but now being used in the backend for this PR. We'd need to keep it on the client to decide what to show in the buddy list when we have full subscriber data

Reply:
Yeah I guess it's fine to just do `subscriber_count < 250` in the query, since that ensures we have all subscribers for channels both when `subscriber count <= 75` and when `realm_count < 250`, and simpler.